docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -1,166 +1,146 @@
+| `api_url`               | Yes      | Endpoint used to obtain user information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |              |
+| `scopes`                | No       | List of comma- or space-separated Okta OAuth2 scopes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `user:email` |
+| `allow_sign_up`         | No       | Controls Grafana user creation through the generic Okta OAuth2 login. Only existing Grafana users can log in with Okta OAuth2 if set to `false`.                                                                                                                                                                                                                                                                                                                                                                               | `true`       |
+| `auto_login`            | No       | Set to `true` to enable users to bypass the login screen and automatically log in. This setting is ignored if you configure multiple auth providers to use auto-login.                                                                                                                                                                                                                                                                                                                                                         | `false`      |
+| `role_attribute_path`   | No       | [JMESPath](http://jmespath.org/examples.html) expression to use for Grafana role lookup. Grafana will first evaluate the expression using the Okta OAuth2 ID token. If no role is found, the expression will be evaluated using the user information obtained from the UserInfo endpoint. The result of the evaluation should be a valid Grafana role (`Viewer`, `Editor`, `Admin` or `GrafanaAdmin`). For more information on user role mapping, refer to [Configure role mapping]({{< relref "#configure-role-mapping" >}}). |              |
```

Comment:
This logic is slightly different for different OAuth providers. Just making a note to double-check if it's the same for Okta as it is for generic OAuth.

Reply:
From the original [documentation](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/okta/#map-roles) the text looks ok. Please let me know if I am missing anything.

Reply:
If it is accurate, then I'm happy to leave it as is 👍